tests/stubtest_allowlists/win32-py312.txt

```diff
@@ -5,11 +5,3 @@ asyncio.windows_events.IocpProactor.finish_socket_func
 asyncio.windows_events.IocpProactor.recvfrom_into
 msvcrt.GetErrorMode
 ntpath.isdir
-os.get_blocking
-os.listdrives
-os.listmounts
-os.listvolumes
-os.path.isdir
```

Comment:
I'd just add this allowlist entry back for now to fix the stubtest error; it's different enough to the others that it's probably good to consider it in its own PR